src/pages/iou/steps/IOUConfirmPage.js

```diff
@@ -7,6 +7,9 @@ const propTypes = {
     /** Callback to inform parent modal of success */
     onConfirm: PropTypes.func.isRequired,
 
+    /** Callback to to parent model to send money */
```

Comment:
NAB: A small typo here.
```suggestion
    /** Callback to parent modal to send money */
```

Reply:
+1 nice catch. except I think not NAB, why do we want to merge something with a typo 😄

Reply:
Haha, you are right, I tend to write the `NAB` to anything which does no break the intended functionality, which is not ideal 😅 I will need to make sure I enforce the conventions more.